### ISSUE TITLE
Consider making data and object property icons distinct

### ISSUE THREAD

**Issue by matthewhorridge** [protegeproject/webprotege#357]
The WebProtege UI currently uses the same icon for data and object property icons.  There has been a request via the mailing list to make them distinct.  See below:

```
Message: 9
Date: Tue, 24 May 2016 17:37:25 +0000
Hi Matthew,

Yes this is in WebProtege.

Rosetta

From: Matthew Horridge <matthew.horridge@stanford.edu>
To: Developer support for WebProtege and Protege Desktop
    <protege-dev@lists.stanford.edu>
Subject: Re: [protege-dev] Different Icon for Data and Object Property
Message-ID: <BEDAACA5-EF75-4297-A8ED-2BD4C37C6A41@stanford.edu>
Content-Type: text/plain; charset="us-ascii"

Hi Rosetta,

Is this in WebProtege?

Cheers,

Matthew


On 24 May 2016, at 03:56, Rosetta.Romano > <Rosetta.Romano@canberra.edu.au<mailto:Rosetta.Romano@canberra.edu.au>> wrote:

Hi,

Would it be possible to have a different icon for the Data and Object Property in the editing interface.

Rosetta
_______________________________________________
protege-dev mailing list
protege-dev@lists.stanford.edu
https://mailman.stanford.edu/mailman/listinfo/protege-dev
```

**Comment by csnyulas** [comment]
I would suggest to keep the current icon ![dataprop_icon1](https://cloud.githubusercontent.com/assets/2441046/15618053/e8d12ab4-2400-11e6-8c8f-f2a204307361.png) for data properties, use the icon that is currently used for annotation properties (![objprop_icon1](https://cloud.githubusercontent.com/assets/2441046/15618066/007c23f8-2401-11e6-9661-01cf33fa7063.png)) for object properties (as the connection image suggest the fact that they express relationships), and I would use something like this ![annprop_icon2](https://cloud.githubusercontent.com/assets/2441046/15618253/d6f096a8-2401-11e6-8b5a-1b09d402b71c.png) for annotation properties.

We could adopt the new icon starting from WebProtege 3.0.

**Comment by Veneranda** [comment]
Hi Matthew,

You asked me to say why I would find separate icons for data properties and object properties in the user interface (UI) useful.  It is because I have developed a large ontology and a colleague wrote some code to identify duplicate label names.  As I am working through the list of duplicates, I can see that I have created both an object and a data property (each with different IRIs).  Actually, in an earlier post to this list you also noted this in your response.  I now understand the difference, and need to make quite a number of corrections.

I can confirm duplicates by using the search results tab in the Class tab, because two lines with the same label appear.  I am also able see the classes where I have assigned the different types of properties by switching on the Usage tab on the Properties tab.  That's all good.    

What would be useful for me (because I have assigned 72 entities to the Data Property, and 95 to the Object Property) is to be able to identify the different type of property by the icon. If this was available then identifying the errors to be corrected would become more obvious, and keystrokes and hovering would be minimised.

Rosetta
